content/docs/reference/providers/pulumi/aws/s3/bucket.md

```diff
@@ -0,0 +1,1827 @@
+---
+title: "Bucket"
+---
+
+<!-- WARNING: this file was generated by the Pulumi Terraform Bridge (tfgen) Tool. -->
+<!-- Do not edit by hand unless you're certain you know what you are doing! -->
+
+<style>
+  table td p { margin-top: 0; margin-bottom: 0; }
+</style>
+
+Provides a S3 bucket resource.
+
+> This content is derived from https://github.com/terraform-providers/terraform-provider-aws/blob/master/website/docs/r/s3_bucket.html.markdown.
+
+
+## Example Usage
+
+### Private Bucket w/ Tags
+
+{{< langchoose nojavascript nogo >}}
+
+```typescript
+import * as pulumi from "@pulumi/pulumi";
+import * as aws from "@pulumi/aws";
+
+const bucket = new aws.s3.Bucket("b", {
+	acl: "private",
+	tags: {
+		Environment: "Dev",
+		Name: "My bucket",
+	},
+});
+```
+
+```python
+Coming soon
+```
+
+### Static Website Hosting
+
+{{< langchoose nojavascript nogo >}}
+
+```typescript
+import * as pulumi from "@pulumi/pulumi";
+import * as aws from "@pulumi/aws";
+import * as fs from "fs";
+
+const bucket = new aws.s3.Bucket("b", {
+	acl: "public-read",
+	policy: fs.readFileSync("policy.json", "utf-8"),
+	website: {
+		errorDocument: "error.html",
+		indexDocument: "index.html",
+		routingRules: `[{
+	"Condition": {
+		"KeyPrefixEquals": "docs/"
+	},
+	"Redirect": {
+		"ReplaceKeyPrefixWith": "documents/"
+	}
+}]
+`,
+	},
+});
+```
+
+```python
```

Comment:
I'd add the other languages here too, even if empty, with a link to the work item so folks know they're on the way.